Pipfile

```diff
@@ -6,27 +6,11 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-astroid = "==2.3.2"
-certifi = "==2019.9.11"
-chardet = "==3.0.4"
-idna = "==2.8"
-isort = "==4.3.21"
-lazy-object-proxy = "==1.4.2"
-mccabe = "==0.6.1"
-oauthlib = "==3.1.0"
-pylint = "==2.4.3"
 python-dotenv = "==0.10.3"
```

Comment:
You can update to last versions of :
PlexAPI==4.4.1
python-dotenv==0.15.0

Reply:
I do prefer keeping one change per one PR. but I guess can make an exception here.